src/hotspot/os/bsd/os_bsd.cpp

```diff
@@ -1401,7 +1401,13 @@ void os::get_summary_cpu_info(char* buf, size_t buflen) {
       strncpy(machine, "", sizeof(machine));
   }
 
-  snprintf(buf, buflen, "%s %s %d MHz", model, machine, mhz);
+  char emulated[16] = "\0";
+#ifdef __APPLE__
+  if (VM_Version::is_cpu_emulated()) {
+    strcpy(emulated, " (EMULATED)");
```

Comment:
I think code checkers will complain about using `strcpy()` here.
I believe that `strncpy()` is preferred.

Reply:
Or perhaps:

const char* emulated = "";
#ifdef __APPLE__
    if (VM_Version::is_cpu_emulated()) {
        emulated = " (EMULATED)";
    }
#endif
...

?

Reply:
That will work very nicely, thank you Mikael!